tests/ntp/test_ntp.py

```diff
@@ -42,13 +55,53 @@ def setup_ntp(ptfhost, duthosts, rand_one_dut_hostname, creds):
     for ntp_server in ntp_servers:
         duthost.command("config ntp add %s" % ntp_server)
 
+@pytest.fixture
+def setup_long_jump_config(duthosts, rand_one_dut_hostname):
+    """set long jump config and set DUT's time forward"""
+
+    duthost = duthosts[rand_one_dut_hostname]
+
+    # get time before set time
+    start_time_dut = int(duthost.command("date +%s")['stdout'])
+    start_time = time.time()
+
+    # stop NTP and set time on DUT
+    duthost.service(name='ntp', state='stopped')
+    duthost.command("date -s '@{}'".format(start_time_dut - TIME_FORWARD))
+
+    # set long jump config with variable
+    yield
+
+    # set DUT's time back after long jump test
+    duthost.service(name='ntp', state='stopped')
+    config_long_jump(duthost)
```

Comment:
Collect NTP enabled/disabled state before test and retore it here?

Reply:
That's better